// tbclient.PbPage.PbPageResIdl
syntax = "proto3";

import "Error.proto";
import "SimpleForum.proto";
import "Page.proto";
import "Post.proto";
import "ThreadInfo.proto";
import "User.proto";

message PbPageResIdl {
    Error error = 1;
    message DataRes {
        SimpleForum forum = 2;
        Page page = 3;
        repeated Post post_list = 6;
        ThreadInfo thread = 8;
        repeated User user_list = 13;
        int32 has_fold_comment = 68;
    }
    DataRes data = 2;
}
